custom_components/tuya_local/devices/madimack_elite_v3_heatpump.yaml

```diff
@@ -117,6 +131,22 @@ secondary_entities:
             value: F
           - dps_val: c
             value: C
+  - entity: sensor

```

Comment:
I don't think this is required, since it is duplicating the "current_temperature" of the climate entity above.

Reply:
This was actually on purpose. In the Tuya app, Inlet and Outlet temperatures are also displayed in the sensor list values, which is quite convenient when adjusting waterflow to get the best efficiency out of the heat pump.

Reply:
Home Assistant is not the Tuya App. You can customise the UI however you want it, that does not depend on duplicating the sensor values.

Reply:
Fair enough, that makes sense. I'll work on the faut codes, see if I can come up with something logical to group different errors. Thank for your patience :-)